src/sagas/projects.js

```diff
@@ -107,11 +107,27 @@ function generateProjectKey() {
 
 export function* projectExported({payload: {exportType}}) {
   if (exportType === 'repo') {
-    const state = yield select();
-    yield call(saveCurrentProject, state);
+    yield* saveCurrentProject();
   }
 }
 
+function* saveCurrentProject() {
+  const state = yield select();
+  const userId = getCurrentUserId(state);
+  const currentProject = getCurrentProject(state);
+
+  if (userId && currentProject && !isPristineProject(currentProject)) {
+    try {
+      yield call(saveProject, userId, currentProject);
+      yield put(projectSuccessfullySaved());
+    } catch (e) {
+      return false;
```

Comment:
I don’t think these `return false`s are meaningful in the context of a saga? Or are there semantics I’m unaware of?

Assuming they aren’t, there is a good question regarding what _should_ go here in the absence of proper error handling (which as previously discussed would make sense to come in a different PR).

Bad pitch, maybe just remove the `try/catch` altogether and let it fail / generate a Bugsnag error?